src/lightning_app/cli/lightning_cli.py

```diff
@@ -206,15 +290,63 @@ def stop():
     pass
 
 
-@_main.group(hidden=True)
+@_main.group(name="delete")
 def delete():
-    """Delete an application."""
+    """Delete a Lightning managed resources."""
```

Comment:
Please change to: 

Delete Lightning BYOC managed resources.

Reply:
:+1: